test/unit/exception_handling_test.rb

```diff
@@ -180,6 +180,20 @@ class SmtpClientErrbackStub < SmtpClientStub
       end
```

Comment:
not related to your change, but I noticed it while looking thorugh this file.. the `#log_info` context just above.. has a test that says `This is an Info` but it's actually calling `log_warning`! Can you change that to a `log_info` and in that test, also confirm the severity is INFO?

Reply:
And.. ditto for `#log_debug` 🙃